test/sosdemo8.jl

```diff
@@ -28,7 +28,8 @@
     P = a + b*x + c*x^2
 
     # Nonnegative on the support
-    @constraint(m, P >= 0, domain = (@set 0 <= x && x <= 5))
+	K = @set 0 <= x && x <= 5 
```

Comment:
here too

Reply:
Why does the documentation still fail?

Reply:
It's not failing, it didn't run because CSDP failed to solve SOSDEMO10. We should move this test to `test/Test` and use mock optimizer on travis to avoid that :)

Reply:
I restarted the failing build (it was Linux v1.1) so you don't see it failing but travis did not run the doc build anyway. I just triggered it